src/thor/bidirectional_astar.cc

```diff
@@ -457,8 +457,8 @@ inline bool BidirectionalAStar::ExpandReverseInner(GraphReader& graphreader,
   // can properly recover elapsed time on the reverse path.
   Cost transition_cost =
       costing_->TransitionCostReverse(meta.edge->localedgeidx(), nodeinfo, opp_edge, opp_pred_edge);
-  Cost newcost = pred.cost() + costing_->EdgeCost(opp_edge, t2, kConstrainedFlowSecondOfDay);
-  newcost.cost += transition_cost.cost;
+  Cost newcost =
+      pred.cost() + costing_->EdgeCost(opp_edge, t2, kConstrainedFlowSecondOfDay) + transition_cost;
```

Comment:
this was an actual change. previously we only added the cost part onto the new cost i guess with the idea that it would be easier to get the cost right in FormPath below when connecting the two search trees together. but there was a bug in that actually and when i fixed it i fixed this as well